src/PrestaShopBundle/Resources/config/services/bundle/event_listener.yml

```diff
@@ -47,32 +47,26 @@ services:
     tags:
       - { name: kernel.event_listener, event: console.command, method: onConsoleCommand }
 
-  prestashop.access_denied.listener:
-    class: PrestaShopBundle\EventListener\AccessDeniedListener
-    arguments:
-      - "@router"
-      - "@translator"
-      - "@session"
+  PrestaShopBundle\EventListener\AccessDeniedListener:
+    autowire: true
     tags:
       - { name: kernel.event_listener, event: kernel.exception, method: onKernelException }
 
+  prestashop.access_denied.listener:
```

Comment:
We can probably remove those service aliases, the listeners are probably not overridden or decorated in any module